packages/liferay-npm-scripts/src/scripts/fix.js

```diff
@@ -9,7 +9,7 @@ const format = require('./format');
 const lint = require('./lint');
 
 function fix() {
-	spawnMultiple(() => format(), () => lint({fix: true}));
+	spawnMultiple(() => lint({fix: true}), () => format());
```

Comment:
I think it won't matter, but should we do the same for the [check](https://github.com/liferay/liferay-npm-tools/blob/master/packages/liferay-npm-scripts/src/scripts/check.js#L12) script?

Reply:
Consistency is good, but the other factor here is which feedback do you think is more important? And once you know that, do you want the important feedback to appear at the beginning or the end?

Reply:
That's an interesting thought... eslint will certainly provide more valuable feedback than prettier during the `check` phase, I guess, which better be the last information we see as it's the most costly to fix? I don't really have a strong opinion here :)

Reply:
> I don't really have a strong opinion here 

Yeah, me neither. 😂 But I think you're right: ESLint provides the higher-signal feedback, so it should probably be last.